pkg/reader/reader_test.go

```diff
@@ -28,10 +28,10 @@ func TestGetVarMap_Required(t *testing.T) {
 
 			for k, v := range varMap {
 				if v.Required && v.Variable.Default != nil {
-					t.Errorf("Variable %s is required but has a default", k)
+					t.Errorf("Variable %q is required but has a default", k)
```

Comment:
V -> v?

Reply:
I generally use capitals in t.Errorf statements. There isn't really any reason for that but I also think it's okay to leave it that way. Mainly because this is 'just the tests'